doc/dev/toolchain-setup.md

```diff
@@ -44,7 +44,7 @@ To build the toolchain, you'll need:
 
 On Ubuntu (including WSL), you can install them with:
 
-``sudo apt get install clang ninja-build lld cmake``
+``sudo apt install clang ninja-build lld cmake``
```

Comment:
While you're at it, it would be nice to change it to this:

````suggestion
```sh
sudo apt install clang ninja-build lld cmake
```
````

Reply:
Good idea